src/Storefront/Page/Account/Login/AccountLoginPageLoader.php

```diff
@@ -88,13 +88,13 @@ private function getSalutations(SalesChannelContext $salesChannelContext): Salut
     private function getCountries(SalesChannelContext $salesChannelContext): CountryCollection
     {
         $criteria = (new Criteria())
-            ->addFilter(new EqualsFilter('active', true))
```

Comment:
was intended to drop the active filter?

Reply:
Yes, that's already default of the `CountryRoute`